src/main.c

```diff
@@ -47,9 +51,14 @@
 unsigned char G_io_seproxyhal_spi_buffer[IO_SEPROXYHAL_BUFFER_SIZE_B];
 esdt_info_t esdt_info;
 
+#ifdef HAVE_BAGL
+void io_seproxyhal_display(const bagl_element_t *element);
+#else
+nbgl_page_t *pageContext;
```

Comment:
Why do you need this?

Reply:
Legacy before some factorization.
`pageContext` is now in `utils.c`